extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java

```diff
@@ -5,13 +5,7 @@
 
 import com.fasterxml.jackson.core.type.TypeReference;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Avoid importing * to prevent collisions - there's an intellij setting where you can set the number of classes from 1 package to use * - you can set it to 99 and intelliJ will never import * again

Reply:
Thanks. I have configured IDE now.